apisix/admin/ssl.lua

```diff
@@ -70,6 +70,17 @@ local function check_conf(id, conf, need_id)
         end
     end
 
+    if conf.client then
+        if not apisix_ssl.support_client_verification() then
```

Comment:
This check is difficult when we using manager-api, unless we have a mechanism to advertise this capability to it (like rolling feature sets in the server-info plugin).

Reply:
There is an additional check in the DP side. The `conf.client` will be ignored if the DP doesn't support it. The check here is not critical as the DP may be different with the Admin API.

Reply:
Got it.